assets/queries/ansible/aws/rds_without_backup/query.rego

```diff
@@ -0,0 +1,46 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  instance := task["community.aws.rds_instance"]
+  instanceName := task.name
+
+  object.get(instance, "backup_retention_period", "undefined") == "undefined"
+
+  result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.rds_instance}}", [instanceName]),
+                "issueType":        "MissingAttribute",
+                "keyExpectedValue": "community.aws.rds_instance should have the property 'backup_retention_period' greater than 0",
+                "keyActualValue":   "community.aws.rds_instance has the property 'password_reuse_prevent' unassigned"
+              }
+}
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  instance := task["community.aws.rds_instance"]
+  instanceName := task.name
+
+  instance.backup_retention_period == 0
+
+  result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.rds_instance}}.backup_retention_period", [instanceName]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue": "community.aws.rds_instance should have the property 'backup_retention_period' greater than 0",
+                "keyActualValue":   "community.aws.rds_instance has the property 'password_reuse_prevent' assigned to 0"
```

Comment:
fix actual value to report backup_retention_period field

Reply:
Fixed